### DISCUSSION TITLE
EDDI 3.73 not loading

### DISCUSSION THREAD

**Discussion by Rendclaw** [EDCD/EDDI#2158] [EDDI Standalone Q&A]
EDDI was working fine until I want to say three nights ago, when I loaded EDDI along with EDDiscovery to start a session, and it worked fine for about an hour then it stopped working. When I tried to restart it, it would not load back in. I uninstalled and reinstalled the program but it is having the same problem.

I'm running ED but without the Odyssey addon. Just Horizons.

**Comment by AkumaSeijin** [comment]
Im having the same exact problem. reinstalled and still nothing. Hoping I didnt uninstall my clone profile.

**Comment by Rendclaw** [comment]
Well I fixed it by deleting everything related to eddi in my windows
folder. It got rid of my clone but that's okay, because I didn't have very
many alterations with it. I deleted everything and reinstalled 3.73 and it
works just fine now. Something was causing the program to crash every time
I tried to start it, although I'm not sure exactly what it was. It sounds
to me like you're worried about having to delete your clone, but that was
the only way I found it to make the program work again. Good luck

**Comment by Tkael** [comment]
I'd guess that there was a journal entry that caused EDDI to crash and that starting a new game session cleared the journal entry causing the crash. If this occurs again, we'd like to examine the logs. In any case, we're glad to hear that you found a way to resolve the issue. o7

**Reply by AkumaSeijin** [reply]
I ended up deleting everything and it worked. Then 2 days later it happened again and this time I ONLY deleted the EDDI folder in %localappdata% and it worked without uninstalling the whole program.

I believe it had something to do with VA crashing but am not 100% sure. After VA crashed is when I couldnt open EDDI by itself.

**Reply by Tkael** [reply]
Interesting. We only store a very small amount of data at that location - the window position and tab selection from EDDI's last run. It sounds like the config file somehow became corrupted.
